packages/browser/scripts/checkbrowsers.js

```diff
@@ -0,0 +1,132 @@
+// Script which checks all browsers in test/integration/browser.js against supported BrowserStack browsers
+// Meant to be run manually, by running `yarn test:integration:checkbrowsers` from the command line
+
+const btoa = require('btoa');
+const fetch = require('node-fetch');
+const localConfigs = require('../test/integration/browsers.js');
+
+const browserstackUsername = process.env.BROWSERSTACK_USERNAME;
+const browserstackAccessKey = process.env.BROWSERSTACK_ACCESS_KEY;
+
+const hasCreds = () => {
+  return browserstackUsername !== undefined && browserstackAccessKey !== undefined;
+};
+
+const fetchCurrentData = (username, key) => {
+  const authKey = btoa(`${username}:${key}`);
+
+  return fetch('https://api.browserstack.com/5/browsers?flat=true', {
+    headers: {
+      Authorization: `Basic ${authKey}`,
+    },
+  }).then(response => {
+    if (response.status >= 200 && response.status < 300) {
+      return response.json();
+    } else {
+      throw new Error(`Unable to fetch data. Status: ${response.status} ${response.statusText}`);
+    }
+  });
+};
+
+const isMatchingEntry = (key, localConfig, bsConfig) => {
+  let localValue = localConfig[key];
+  let bsValue = bsConfig[key];
+
+  // all values are either null, undefined, or strings, so this should only catch strings
+  if (localValue) {
```

Comment:
We don't care about nulls or undefined, as they are treated as equal, so we can coerce them as well.

This whole check can be transformed to:

```js
if (`${localValue}`.toLowerCase() === `${bsValue}`.toLowerCase()) return true;
```
```

Reply:
I don't think so. If you try to lowercase `null` or `undefined`, you get an error:

![image](https://user-images.githubusercontent.com/14812505/74741001-4cf7d280-525c-11ea-96bd-c129a7b7b84c.png)

Also, `null` and `undefined` are only equal in the non-strict sense. Using the same `x` and `y` as above:

![image](https://user-images.githubusercontent.com/14812505/74741067-74e73600-525c-11ea-88df-e58833729a52.png)

However, I do think the point of my comment could be clearer. New version:

```
  // all values are either null, undefined, or strings, so checking truthiness should
  // save us from trying to lowercase anything that can't handle it
  if (localValue) {
    localValue = localValue.toLowerCase();
  }
  if (bsValue) {
    bsValue = bsValue.toLowerCase();
  }
```